README.md

```diff
@@ -28,10 +31,12 @@ We mount the `/home/test` folder to the running image to make files available to
 The command line parameters after the image name are passed to the software inside the image.
 Note that the paths provided are relative to the running image (see the --mount option specified above).
 
-- `--working_space "/mnt"` specifies the folder to use as a workspace. Contains the .ply files to convert
+- `--working_space "/mnt"` specifies the folder to use as a workspace
 - `--metadata "/mnt/08f445ef-b8f9-421a-acf1-8b8c206c1bb8_metadata.cleaned.json"` is the name of the source metadata to be cleaned
+- `/mnt/3c807fe1-a5ba-4b4b-b618-1d2c9c981678__Top-heading-east_0.ply` is the name of the ply file to convert
 
 # Original README
```

Comment:
Why include this separately as "Original README" rather than integrating the existing material into the new README (deleting obsolete content and keeping what is relevant)? Pinging @Sithyphus b/c porting README's is on his todo list

Reply:
I was thinking that someone might find it useful? I originally kept it out but then I put it in even though it's not useful IMO - I have no strong opinion.

Reply:
Creating ticket to get the original README better integrated into the new README (instead of a separate section of the README)